tests/test_unit/test_transform_utils.py

```diff
@@ -0,0 +1,50 @@
+import dask.array as da
+import numpy as np
+import pytest
+from skimage import transform
+
+from brainglobe_template_builder.preproc.transform_utils import (
+    downsample_anisotropic_image_stack,
+)
+
+
+@pytest.fixture()
+def stack():
+    """Create a dask array representing an image stack"""
+    data = np.random.rand(10, 100, 100)  # Random image stack
```

Comment:
sonarlint complains about using the legacy random functions. The new generator should be used insted.
Just mentioning it here, in case you care.